pkg/fakerp/fakerp.go

```diff
@@ -201,6 +203,15 @@ func createOrUpdateWrapper(ctx context.Context, p api.Plugin, log *logrus.Entry,
 		return nil, err
 	}
 
+	if !isUpdate {
```

Comment:
@amanohar the real RP would have to do something similar (create the workspace before creating the cluster)